src/tabs/tests.rs

```diff
@@ -0,0 +1,38 @@
+use super::*;
```

Comment:
how about putting this into `/tests/tabs.rs` instead?

Reply:
This is also a personal taste. I prefer to have most tests close to the actual code.